### PR TITLE
adding support for automatic updates

### PR DESCRIPTION
**Pull request by alexanderpaolini**
- Added support for fetching `https://shiro.gg/api/endpoints`

### CODE REVIEW

File: src/worker.js [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
-
-nsfw.forEach(type => {
-  worker.commands.add({
-    command: type,
-    image: true,
-    nsfw: true
+fetch('https://shiro.gg/api/endpoints')
+  .then(res => res.json())
+  .then(({ nsfw, sfw }) => {
+    nsfw = nsfw.map(endpoint => endpoint.slice(12))
```
Comment (jpbberry) [inline src/worker.js]: make it split by slash then get the last one rather than slicing.

**Review by jpbberry — CHANGES_REQUESTED** [review]

File: src/worker.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ const worker = new Worker()
 fetch('https://shiro.gg/api/endpoints')
   .then(res => res.json())
   .then(({ nsfw, sfw }) => {
-    nsfw = nsfw.map(endpoint => endpoint.slice(12))
-    sfw = sfw.map(endpoint => endpoint.slice(7))
+    nsfw = nsfw.map(endpoint => endpoint.split('/')[2])
```
Comment (jpbberry) [inline src/worker.js]: use pop

**Review by jpbberry — CHANGES_REQUESTED** [review]

File: src/worker.js (hunk 1 above)
Comment (alexanderpaolini) [reply src/worker.js]: done bro

File: src/worker.js (hunk 2 above)
Comment (alexanderpaolini) [reply src/worker.js]: Fixed in latest commit.

**Pull request merged**